extensions/notebook/package.nls.json

```diff
@@ -3,7 +3,7 @@
 	"description": "Defines the Data-procotol based Notebook contribution and many Notebook commands and contributions.",
 	"notebook.configuration.title": "Notebook configuration",
 	"notebook.pythonPath.description": "Local path to python installation used by Notebooks.",
-	"notebook.sqlKernelEnabled.description": "Enable SQL kernel in notebook editor (Preview)",
+	"notebook.sqlKernelEnabled.description": "Enable SQL kernel in notebook editor (Preview). Requires restarting Azure Data Studio on change",
```

Comment:
Would this be better worded as `result in ADS reload`? How its currently worded kind of sounds like the user will have to do something.

Reply:
Good catch. Reload is what we use for extensions, will update.